src/code/code_80097A00.c

```diff
@@ -224,30 +225,43 @@ u8 gItemSlots[] = {
     SLOT_TRADE_ADULT, SLOT_TRADE_ADULT,
 };
 
-void Inventory_ChangeEquipment(s16 equipment, u16 value) {
-    gSaveContext.equips.equipment &= gEquipNegMasks[equipment];
-    gSaveContext.equips.equipment |= value << gEquipShifts[equipment];
+/**
+ * Change currently-equipped item
+ * 
+ * @param type Type of equipment to change. Use `EquipmentType` enum
+ * @param value Value of equipment to change. Use the appropriate `EquipValue____` enum
+ */
+void Inventory_ChangeEquipment(s16 type, u16 value) {
+    gSaveContext.equips.equipment &= gEquipNegMasks[type];
+    gSaveContext.equips.equipment |= value << gEquipShifts[type];
 }
 
-u8 Inventory_DeleteEquipment(PlayState* play, s16 equipment) {
+/**
+ * Deletes the currently-equipped equipment of type `equipmentType`
+ * 
+ * @param equipmentType Type of equipment (sword/shield/tunic/boots) to delete (uses `EquipmentType` enum).
+ * 
+ * @return u8 deleted item
```

Comment:
Having the type seems redundant?

Reply:
Interestingly this is what the default Doxygen generator does.